Give-Hope-org/give-hope-fn
# register component both for individuals and organizations

**What does this PR do?**

This PR does the following:

- Adds a header navbar to the signup form for individuals and organizations to give users access to routes to other pages.
- Fixes the fixed navbar bug that was hiding the signup form.
- Changes the login illustration to a more relevant one.
- Hides the illustration on mobile for better user experience.

**Description of Task to be Completed?**    

 the PR includes screenshots of the changes made on large screens, landscape phones, and tablets. These screenshots provide visual confirmation of the changes made and help the reviewer to understand the impact of the changes on different devices.      

**How should this be manually tested?**   
 
- navigate to the register route  for individuals and organization
- check the responsiveness  on different screens (mobile, laptop, tablet, large screens)     

**Any background context you want to provide?**

None   

Screenshots :


Organization :

 Large screens :

![Tablet](https://www.linkpicture.com/q/pc_6.png)

Tablet :

![Tablet](https://www.linkpicture.com/q/tablet_2.png)

mobile :

![Mobile](https://www.linkpicture.com/q/mobile_4.png)
